### PR TITLE
fix(style): use font in dialog

### LINKED ISSUE #534: Wrong fonts used in dialogs
**Describe the bug:**

Default fonts are used in dialogs

**To Reproduce:**
image

**Expected behavior:**

Should use font from application 

**Additional context:**

### PR DESCRIPTION
**Pull request by DanielHabenicht**
fix #534

### CODE REVIEW

File: Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="displayDialog">
-  <h3 mat-dialog-title class="pb-color-primary" i18n="Display-notificationDialog|Welcome Headline@@WelcomeHeadline">
+  <h1 mat-dialog-title class="pb-color-primary" i18n="Display-notificationDialog|Welcome Headline@@WelcomeHeadline">
```
Comment (paule96) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]: h2

Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]:
```suggestion
  <h2 mat-dialog-title class="pb-color-primary" i18n="Display-notificationDialog|Welcome Headline@@WelcomeHeadline">
```

File: Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
     Welcome to the new Phonebook!
-  </h3>
+  </h1>
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html]:
```suggestion
  </h2>
```

**Comment by mschwrdtnr** [comment]
@DanielHabenicht can i now update the imagetag?

**Comment by DanielHabenicht** [comment]
Yep

**Pull request merged**